Lib/collections/abc.py

```diff
@@ -1,3 +1,5 @@
 from _collections_abc import *
 from _collections_abc import __all__
 from _collections_abc import _CallableGenericAlias
+
+__all__ = [*__all__, '_CallableGenericAlias']
```

Comment:
I do not think that `_CallableGenericAlias` should be imported by the star import.